README.md

```diff
@@ -160,12 +174,11 @@ mremi_contact:
         name:              contact_form
         validation_groups: [Default]
         subject_provider:  mremi_contact.subject_provider.noop
-        captcha_disabled:  false
-        captcha_type:      genemu_captcha # or genemu_recaptcha
+        captcha_type:      false
```

Comment:
Yes because the default value will be `NULL` unless you configure it, and this snippet shows the default values. This is useful for a quick start when you install the bundle.
I think a dedicated snippet after default values should be more appropriate and allows to highlight the captcha config.